packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/en.ftl

```diff
@@ -37,8 +37,8 @@ tfa-input-enter-totp-v2 =
 tfa-save-these-codes-1 = Save these one-time use backup authentication codes in a safe place for when
   you don’t have your mobile device.
 
-tfa-enter-code-to-confirm-v2 = Please enter one of your new backup authentication codes to
```

Comment:
This string was in the wrong file. I moved it to Page2faReplaceRecoveryCodes and added an entirely missing string.